charts/argo-cd/templates/argocd-server/clusterrole.yaml

```diff
@@ -21,6 +21,9 @@ rules:
       - events
     verbs:
       - list
+      {{- if .Values.configs.params.application.namespaces }}
+      - create
+      {{- end }}
```

Comment:
Since [upstream](https://github.com/argoproj/argo-cd/blob/master/manifests/cluster-rbac/server/argocd-server-clusterrole.yaml#L18-L23) doesn't have `create` in default, I added it with condition for in case. 🤔